script.skinshortcuts/resources/lib/simpleeval.py

```diff
@@ -0,0 +1,313 @@
+'''
+SimpleEval - (C) 2013/2015 Daniel Fairhead
```

Comment:
please move this to a script.module

Reply:
license is also different from the one of your main addon.

Reply:
Is this really necessary? Seems silly to add a seperate module to maintain.

Reply:
Yeah that's the rule but you might do it later, as long as you actually do it :)